Update custom operators URL

crahrig opened the pull request:
Appending an "s" to fix a broken URL path for the "Custom operators" heading link

opbld33 commented:
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/get-started-uwp.md](https://github.com/MicrosoftDocs/windows-ai-docs/blob/master/docs/get-started-uwp.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/windows/ai/get-started-uwp?branch=pr-en-us-22) |
[docs/train-model-vs-tools-ai.md](https://github.com/MicrosoftDocs/windows-ai-docs/blob/master/docs/train-model-vs-tools-ai.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C9%5C17%5C28ddff79-5366-c51c-8202-1b9e8beacb23%5CPullRequest%5C201809171813417075-22%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=JQkoFSX1HfVTfYJl%2FIY9v%2FdVPy%2FUdvrE%2BwTA0DCxGmg%3D&st=2018-09-17T18%3A09%3A55Z&se=2018-10-18T18%3A14%3A55Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

crahrig committed:
Update custom operators URL

Appending an "s" to fix a broken URL path for the "Custom operators" heading link
